src/encoded/static/components/cart/batch_download.js

```diff
@@ -65,11 +65,12 @@ CartBatchDownloadButton.propTypes = {
     /** True if button disabled */
     disabled: PropTypes.bool,
     /** Called when the user clicks the button; provided by `BatchDownloadActuator` */
-    onClick: PropTypes.func.isRequired,
+    onClick: PropTypes.func,
 };
 
 CartBatchDownloadButton.defaultProps = {
     disabled: false,
+    onClick: null, // Actually required; provided by BatchDownloadActuator
```

Comment:
the comment is confusing. The code tell me it's not required but the comment says it is just provided by someone else...